src/sidebar/components/user-menu.js

```diff
@@ -50,6 +60,11 @@ function UserMenu({ auth, bridge, onLogout, serviceUrl, settings }) {
     !isThirdParty || serviceSupports('onProfileRequestProvided');
   const isLogoutEnabled =
     !isThirdParty || serviceSupports('onLogoutRequestProvided');
+  const isNotebookEnabled = features.flagEnabled('notebook_launch');
+
+  const onSelectNotebook = () => {
+    frameSync.showNotebook();
```

Comment:
There are several other menu items in `UserMenu` that just call `bridge.call` directly and I think it would be fine to do the same for showing the notebook. The `frameSync` service's main responsibility is syncing annotations between the sidebar and other frames, I don't think it necessarily needs to wrap all of the cross-frame calls.

Reply:
Ah, gotcha. Makes sense.